guides/common/modules/proc_creating-a-local-file-repository.adoc

```diff
@@ -14,14 +14,25 @@ To create a file type repository in a local directory, complete the following pr
 
 . Ensure the Server and {project-client-name} repositories are enabled.
 ifdef::satellite[]
+** For RHEL 8:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# subscription-manager repos --enable={RepoRHEL8BaseOS} \
+--enable={RepoRHEL8AppStream} \
+--enable={RepoRHEL8ServerSatelliteUtils}
+----
+** For RHEL 7:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # subscription-manager repos --enable={RepoRHEL7Server} \
---enable={project-client-RHEL7-url}
+--enable={RepoRHEL7ServerSatelliteUtils}
 ----
 endif::[]
+ifdef::satellite,orcharhino[]
 . Install the Pulp Manifest package:
+** For RHEL 8:
```

Comment:
I'd drop this line since there's no list or anything.
```suggestion
```

Reply:
Steps are separated here for RHEL 7 and 8. Hence this bullet points are used.